plugin/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "dev": "webpack --config webpack.dev.js",
     "prod": "webpack --config webpack.prod.js",
-    "start": "yarn dev -- --watch",
+    "start": "yarn dev --watch",
```

Comment:
deprecation warning in my build console that yarn hasn't needed this since yarn v1.0